server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAnalysesList.js

```diff
@@ -109,7 +111,7 @@ export default class ProjectActivityAnalysesList extends React.PureComponent {
     }
   };
 
-  updateStickyBadges = (forceBadgeAlignement?: boolean) => {
+  updateStickyBadges = (forceBadgeAlignement /*: ?boolean */) => {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this function to reduce its Cognitive Complexity from 26 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3776)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)